tests/test_version_range.py

```diff
@@ -209,6 +211,32 @@ def test_PypiVersionRange_raises_ivr_for_invalid_ranges(self):
         except InvalidVersionRange as ivre:
             assert str(ivre).startswith("Unsupported character")
 
+    def test_NpmVersionRange_from_native(self):
+        """
+        Tests for NpmVersionRange.from_native
+        """
```

Comment:
This information is already conveyed by the test name. Maybe a different or no docstring ?